### DISCUSSION TITLE
Failed to run backend app

### DISCUSSION THREAD

**Discussion by nmetifiot** [Project-Books/book-project#736] [Q&A] [answered]
Hello, I followed all step to run the frontend and backend app. I successfully run the frontend but I got an error for the backend : 
crbug/1173575, non-JS module files deprecated.
Does anyone knows how to solve it ?

**Comment by knjk04** [comment]
Hi, a few questions:
- How did you run the backend (e.g. via Maven or the terminal)? 
- What JDK are you running (e.g. Oracle or Open) and what's the version number?
- What operating system?

I haven't come across this before, but I'll try to reproduce this error

**Reply by nmetifiot** [reply]
Hello,

I run it by Edge Launch on VS code. This is the configuration : 
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch Edge",
            "request": "launch",
            "type": "pwa-msedge",
            "url": "http://localhost:8080",
            "webRoot": "${workspaceFolder}"
        }
    ]
}

I am using the version number : openjdk version "14.0.2" 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-46)
OpenJDK 64-Bit Server VM (build 14.0.2+12-46, mixed mode, sharing)
On Mac

When I use Itellij I got this error : https://github.com/Project-Books/book-project/discussions/738

Thanks for your help :)

**Reply by nmetifiot** [reply]
Finaly I found how to solve it thanks to this post : https://stackoverflow.com/questions/46671308/how-to-create-a-java-maven-project-that-works-in-visual-studio-code#:~:text=Open%20the%20Maven%20project%20folder%20in%20VS%20Code,from%20template.%20Choose%20maven%20%28%22Executes%20common%20Maven%20commands%22%29.

Know I only have this build error : https://github.com/Project-Books/book-project/discussions/738

**Reply by knjk04** [reply]
Glad to hear you resolved it. Would it be possible to add this as a new answer so that we can mark this as answered?

**Comment by nmetifiot** [comment, marked as answer]
Finaly I found how to solve it thanks to this post : https://stackoverflow.com/questions/46671308/how-to-create-a-java-maven-project-that-works-in-visual-studio-code#:~:text=Open%20the%20Maven%20project%20folder%20in%20VS%20Code,from%20template.%20Choose%20maven%20%28%22Executes%20common%20Maven%20commands%22%29.
